src/LiveDevelopment/LiveDevelopment.js

```diff
@@ -868,6 +873,11 @@ define(function LiveDevelopment(require, exports, module) {
             var closeDeferred = (brackets.platform === "mac") ? NativeApp.closeLiveBrowser() : $.Deferred().resolve();
             closeDeferred.done(function () {
                 _setStatus(STATUS_INACTIVE, reason || "explicit_close");
+                // clean-up registered servers
+                _regServers.forEach(function (server) {
+                    LiveDevServerManager.removeServer(server);
```

Comment:
We don't close servers when the LivePreview session closes. Perhaps, it would be the right moment to close the server.  This, though, will require exposing `closeServer` in at least `StaticServer`... It seems like the node domain command is there, but it's never used. I'll file a separate issue.